packages/blocks/src/page-block/widgets/drag-handle/drag-handle.ts

```diff
@@ -358,16 +353,16 @@ export class AffineDragHandleWidget extends WidgetElement<
     }
     dragPreview.classList.add('grabbing');
     dragPreview.style.display = 'block';
-    this.pageBlockElement.appendChild(dragPreview);
+    document.body.appendChild(dragPreview);
```

Comment:
Is this something that needs to be refactored for supporting multiple editors?

Reply:
not an issue for multiple editors as there can be only one dragPreview globally...
but I agree that adding elements on document.body should be avoided.... I left this by mistake while debugging incorrect dragPreview position due to container offset